spec/rspec/expectations/extensions/kernel_spec.rb

```diff
@@ -1,79 +0,0 @@
-RSpec.describe Object, "#should" do
-  before(:example) do
-    @target = "target"
-    @matcher = double("matcher")
-    allow(@matcher).to receive(:matches?).and_return(true)
-    allow(@matcher).to receive(:failure_message)
-  end
-
-  it "accepts and interacts with a matcher" do
-    expect(@matcher).to receive(:matches?).with(@target).and_return(true)
-    expect(@target).to @matcher
-  end
-
-  it "asks for a failure_message when matches? returns false" do
-    expect(@matcher).to receive(:matches?).with(@target).and_return(false)
-    expect(@matcher).to receive(:failure_message).and_return("the failure message")
-    expect {
-      expect(@target).to @matcher
-    }.to fail_with("the failure message")
-  end
```

Comment:
Did you find duplicates of these? I thought you were moving them (if so a comment saying "these are duplicates of file:line" etc is helpful when reviewing).